packages/lodestar/src/api/rest/controllers/beacon/blocks/getBlockHeaders.ts

```diff
@@ -7,7 +7,7 @@ export const getBlockHeaders: ApiController<{slot?: string; parent_root?: string
 
   handler: async function (req, resp) {
     let slot: Slot|undefined;
-    if(req.query.slot) {
+    if(req.query.slot || req.query.slot == "0") {
```

Comment:
Is this needed? `Boolean("0") === true`
Also seems like something that fastify schema validation should find?

If it is, should use ===

Reply:
> Also seems like something that fastify schema validation should find?

slot 0 is valid, and we are suppose to receive slots as string.
Used == as it would catch both number zero and string zero